packages/cli/src/config/config.ts

```diff
@@ -412,6 +418,7 @@ export async function loadCliConfig(
         process.env.OTEL_EXPORTER_OTLP_ENDPOINT ??
         settings.telemetry?.otlpEndpoint,
       logPrompts: argv.telemetryLogPrompts ?? settings.telemetry?.logPrompts,
+      outfile: argv.telemetryOutfile,
```

Comment:
what if it's set in settings instead or args?

Reply:
now supports settings